src/main/java/org/zalando/nakadi/domain/EventPublishingStep.java

```diff
@@ -4,6 +4,7 @@
     NONE,
     VALIDATING,
     ENRICHING,
+    VALIDATING_SIZE,
```

Comment:
I guess I could either check the size during validation, in which case we ignore whatever is added by the enrichment step; or, go back to validation after enriching, but that may be very confusing to users